core-java-modules/core-java-streams-simple/src/main/java/com/baeldung/streams/minmaxbygroup/OrderItem.java

```diff
@@ -0,0 +1,39 @@
+package com.baeldung.streams.minmaxbygroup;
+
+public class OrderItem {
+  private Long id;
+  private Double price;
+  private OrderItemCategory category;
+
+  // other fields
```

Comment:
let's remove the comment - not necessary

Reply:
Done